tests/unit/services/cookies-test.js

```diff
@@ -358,6 +382,20 @@ describe('CookiesService', function() {
         expect(this.subject().read(COOKIE_NAME)).to.eq(value);
       });
 
+      it('URI-component-decodes the value', function() {
+        let value = '!"§$%&/()=?"';
+        this.subject().write(COOKIE_NAME, value);
```

Comment:
I think you'd explicitly want to write the encoded value here - similar to https://github.com/simplabs/ember-cookies/pull/134/files#diff-91737467d778614edb8e37696092b40bR92

Reply:
Why is that necessary if `write()` is already URI-encoding the value by default? Are you saying that I should pass `encodeURIComponent(value)` to the function just to ensure that it's encoded?

Reply:
I'd say you don't want to go through `write` at all here as that makes the test depend on the correctness of `write` and also a bit harder to understand. You can simply use `document.cookie` directly as in https://github.com/simplabs/ember-cookies/pull/134/files/9392c6d2504835c1bce10ff1503f63e241672972#diff-91737467d778614edb8e37696092b40bR101

Reply:
I'm using `document.cookie` directly in the regular (non-Fastboot) context. But all of the other unit tests for reading Fastboot cookies involve calling the `write()` function to write them. Should those be changed too? 🤔

Reply:
Ah sorry, this is a FastBoot test of course 🤦‍♂️ Yes, I do thing we should change those tests to set the header directly instead of relying on `write` but that's something for another PR.